### PR TITLE
Changes in the teflo-examples repo

### PR DESCRIPTION
**Pull request by rujutashinde**
* Corrected provision and orchestrate examples to use supported keys
* Modified readmes for differnt examples
* Corrected urls
* Removed run.sh script to run teflo scenarios
* Added new examples for exeute and data ass thru for orchestrate

### CODE REVIEW

File: configuration/teflo/teflo.cfg [hunk 1]
```diff
@@ -31,25 +32,88 @@ resource_check_endpoint=<endpoint_ url_for_dependency_check>
...
 
-# credentials settings
+# credentials file and Ansible Vault password
+#
+# User can set all the credential information in a text file and encrypt it using ansible vault.
+# So that way users do not have to provide any credentials in the teflo.cfg file
+# Users then have to provide the path of the encrypted file under CREDENTIALS_PATH.
+# Provide the vault password as VAULTPASS. This password can be
+# exported as an environmental variable
+CREDENTIAL_PATH=<path for the credentials txt file>
```
Comment (JunqiZhang0) [inline configuration/teflo/teflo.cfg]: Doesn't have to be txt

File: e2e/template.yml [hunk 2]
```diff
@@ -12,71 +11,99 @@ description: >
...
 #
-# internally, most likely Semaphore will be used:
 # in teflo.cfg add:
 # [defaults]
-# resource_check_endpoint=https://semaphore.cloud.upshift.engineering.redhat.com/api/v1
-
-# the following is a list of some resources that Semaphore
-# allows you to check:
+# resource_check_endpoint=<endpoint url>
+# Currently only semaphore and statuspage.io is supported
```
Comment (JunqiZhang0) [inline e2e/template.yml]: statuspage.io is not supported yet

File: e2e/template.yml [hunk 3]
```diff
@@ -194,7 +228,7 @@ execute:
         name: tests.sh arg1 arg2
     artifacts: retraint-*, test.log
 
-  - name: test suite 03
+  - name: test_suite_03
```
Comment (JunqiZhang0) [inline e2e/template.yml:231]: Why change here?

File: execute/runner/shell/README.rst [hunk 4]
```diff
@@ -39,14 +37,17 @@ teflo command.
...
-an easy way showing how to run teflo (for a demo purpose).
+To skip the first execute tasks and run only the second one
 
-You can run the script to execute as follows:
 
 .. code-block:: none
 
-    (teflo) $ ./run.sh
\ No newline at end of file
+    (teflo) $ teflo run -t execute -s scenario.yml -sl exe1
```
Comment (JunqiZhang0) [inline execute/runner/shell/README.rst:53]: is exe1 the second one?

File: provision/README.rst [hunk 5]
```diff
@@ -2,19 +2,29 @@ Provision
...
 
-Please see the openstack `readme <openstack>`_ for more details.
+Please see the openstack `readme <linchpin>`__ for more details.
+
+OpenStack_Libcloud
+------------------
+
+Please see the openstack `readme <openstack_libcloud>`__ for more details.
+
+OpenStack_Cliet
```
Comment (JunqiZhang0) [inline provision/README.rst]: typo

File: execute/runner/shell/README.rst (hunk 4 above)
Comment (rujutashinde) [reply execute/runner/shell/README.rst:53]: I am skipping label -sl exe1 here, so exe2 will run

File: provision/README.rst (hunk 5 above)
Comment (rujutashinde) [reply provision/README.rst]: will change

File: e2e/template.yml (hunk 3 above)
Comment (rujutashinde) [reply e2e/template.yml:231]: no reason can keep it the same. But  i think having no spaces is better

File: e2e/template.yml (hunk 2 above)
Comment (rujutashinde) [reply e2e/template.yml]: will change

File: configuration/teflo/teflo.cfg (hunk 1 above)
Comment (rujutashinde) [reply configuration/teflo/teflo.cfg]: will change

File: provision/README.rst (hunk 5 above)
Comment (rujutashinde) [reply provision/README.rst]: done

File: e2e/template.yml (hunk 2 above)
Comment (rujutashinde) [reply e2e/template.yml]: done

File: configuration/teflo/teflo.cfg (hunk 1 above)
Comment (rujutashinde) [reply configuration/teflo/teflo.cfg]: done

**Review by JunqiZhang0 — APPROVED** [review]
lgtm

**Pull request merged**